api/jobset/v1alpha2/jobset_types.go

```diff
@@ -40,12 +40,9 @@ const (
 	NamespacedJobKey        string = "alpha.jobset.sigs.k8s.io/namespaced-job"
 	NoScheduleTaintKey      string = "alpha.jobset.sigs.k8s.io/no-schedule"
 
-	// LabelManagedBy is used to indicate the controller or entity that manages an JobSet
-	LabelManagedBy = "alpha.jobset.sigs.k8s.io/managed-by"
-
-	// JobSetManager is used as the value for LabelManagedBy to identify the jobset controller manager
-	// as the manager of a specific JobSet.
-	JobSetManager = "jobset"
+	// JobSetControllerName is the reserved value for the managedBy field for the built-in
+	// JobSet controller.
+	JobSetControllerName = "jobset.sigs.k8s.io/jobset-controller"
```

Comment:
Should we to introduce validation in the webhook with `IsDomainPrefixedPath`, analogously as for Job: https://github.com/kubernetes/kubernetes/blob/f4e246bc93ffb68b33ed67c7896c379efa4207e7/pkg/apis/batch/validation/validation.go#L220-L223. Potentially also length.

Reply:
Yes I think so, we have been bitten before by not having strict enough validation, so I'd like to enforce the field is restricted to expected format and length

Reply:
I updated the code. Please resolve if nothing else is needed.

Reply:
Can you add a unit tests ensuring the validation works as expected

Reply:
Done.